### DISCUSSION TITLE
Add Support for Magicsee G7 pro

### DISCUSSION THREAD

**Discussion by sunilprregmi** [devmfc/debian-on-amlogic#97] [General]
firmware and other info can be found here

https://slimboxtv.ru/g7-pro/

**Comment by devmfc** [comment]
It would be great if you can make that box work! Please let me know when you succeed and I will add the box to these images.
More info can be found here:

[https://google.com](https://google.com)

Thanx!

**Comment by sunilprregmi** [comment]
can you give me some ideas to build own version of dtb and other things? i am unable to find any of those guides.. but i want to make support for my tv boxes
